website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,245 @@
+A &mdash; [Invocation History list](#invocation-history-list)<br />
+B &mdash; [Invocation Summary](#invocation-summary)<br />
+C &mdash; [System Logs toggle](#system-logs-toggle)<br />
+D &mdash; [Result Status tab](#result-status-tab)<br />
+E &mdash; [Node result](#node-result)<br />
+F &mdash; [Command Control button](#command-control-button)<br />
+
+### Invocation History list
+
+The Invocation History List in the left hand panel of the Invocation History Drawer shows summary details about previous invocations in the IDE. This includes the command, the branch name, the status of the command, and the total elapsed time. 
+
+### Invocation Summary
+
+The Invocation Summary shows the summary details of the selected invocation from the Command History List. This includes the command invoked, the status of the selected command (`Running` if the command is still in progress), the active git branch at the time of the invocation, and the timestamp of the invocation.
+
+### System Logs toggle
+The System Logs Toggle allows the user to see the full stdout and debug logs for entirety of the invoked command. 
+
+### Result Status tab
+The Results Status Tabs group executed nodes from the command by their result. Clicking on these will filter the Node Status List by the corresponding status. Statuses include:
+
+- Pass - successful invocation of a node
+- Warn (tests only) - test executed with warning
+- Error - database error when executing node or failure in test
+- Skip - nodes that did not run due to upstream error
+- Queued - nodes that have yet to execute
+
+### Node result
+Each node executed during a dbt command invocation will be summarized in a Node Result toggle, that contains the summary (stdout) and debug logs to easily see the details about each node. The Node Results List contains an entry for every invoked node. 
+
+### Node result list
+
+The Node result list contains the full list of each Node Result invoked during the dbt run. This can be filtered by clicking on any of the Result Status tabs. 
+### Command Control button
+Use the command control button to control your invocation and cancel or rerun a selected run.
+
+## Modals and Menus
+
+### File Search
+The File Navigation Menu (`CMD/CTRL + O` or the 🔍 icon in the File Explorer) allows for quick searching files by name, and navigating between then with ease.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-file-search.jpg" width="100%" title="The Command History returns a log and detail of all your dbt Cloud invocations."/>
+
+### Global Command Palette
+The Global Command Palette contains useful shortcuts for interacting with the IDE, including git actions, specialized dbt commands, as well as compile and preview actions, among others. Use `CMD/CTRL + P` to access the menu.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-global-command-palette.jpg" width="100%" title="The Command History returns a log and detail of all your dbt Cloud invocations."/>
+
+### IDE Status modal
```

Comment:
You should link from here to #### IDE Status Button to say "Here's how you open this!"